Packs/Jq/Scripts/Jq/Jq.py

```diff
@@ -0,0 +1,35 @@
+import json
+import traceback
+
+import demistomock as demisto  # noqa: F401
+import pyjq
+from CommonServerPython import *  # noqa: F401
+
+
+''' MAIN FUNCTION '''
+
+
+def jq_wrap(json_str, query):
+    if type(json_str) == str:
+        j = json.loads(json_str)
+    else:
+        j = json_str
+
+    res = pyjq.all(query, j)
+    try:
+        demisto.executeCommand('Set', {'key': 'jq.result', 'value': res})
+    except:
+        pass  # ignore issue when can't set context - script executed as transform script
```

Comment:
It is better to get the result to a variable (for example `excute_command_result`) and check with `is_error` if it raised an error (for example `if is_error(excute_command_result):`).